settingUp.md

```diff
@@ -0,0 +1,224 @@
+# If starting from scratch (not cloning repo)
+
+Set up the project using the standard django method
+
+```
+mkdir museum_of_dreams_project
+cd museum_of_dreams_project
+
+python3 -m venv modvenv
+source modvenv/bin/activate
+pip install django
+
+django-admin startproject mod-app
+cd mod-app
+python manage.py runserver
+```
+
+For AWS, you should create a `.ebextensions` folder in the top level of the project (it should be same level as `manage.py` and `requirements.txt`). In this folder create `01_packages.config` with the following contents:
+
+```
+packages:
+  yum:
+    mariadb105-devel: []
+```
+
+This will address issues with setting up the mysql db.
+
+Also create `db-migrate.config` with the following contents:
+
+```
+container_commands:
+  01_migrate_and_collectstatic:
+    test: "[ -f /var/app/current/manage.py ]"
+    command: "python manage.py migrate && python manage.py collectstatic --no-input && echo db-migrate has run"
+    leader_only: true
+    ignoreErrors: true
+option_settings:
+  aws:elasticbeanstalk:application:environment:
```

Comment:
I don't think we need this amount of detail since .ebextensions, settings.local etc are already in the repo. It means if you make changes to these, you'll need to remember to update this file - Don't Repeat Yourself! :-)

Reply:
good point